addons/account_fleet/models/fleet_vehicle_log_services.py

```diff
@@ -1,6 +1,6 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, fields, models, _
+from odoo import Command, _, api, fields, models
```

Comment:
```suggestion
from odoo import  _, api, Command, fields, models
```
alphanumeric ordering

Reply:
Shouldn't it be ASCIIbetically ordered?